src/tasks/setup.ts

```diff
@@ -243,4 +243,11 @@ export const SETUP_TASKS: BaggoTask[] = [
     completed: () => have($item`tiny black hole`),
     limit: { tries: 1 },
   },
+  {
+    name: "Configure Vykea",
+    ready: () => get("_VYKEACompanionLevel") === 0 && bestVykeaLevel() > 0,
```

Comment:
Could get rid of the _VYKEACompanionLevel check here since it is used in .completed